doc/ChangeLog

```diff
@@ -1,4 +1,193 @@
 ===============================================================
+Tag name:  ctsm1.0.dev080
+Originator(s):  slevis (Samuel Levis,SLevis Consulting LLC,303-665-1310)
+Date:  Sun Nov 10 17:11:27 MST 2019
+One-line Summary: Replace masked with "nomask" SRC files for use with mkmapdata and mksurfdata_map
```

Comment:
I'm interested in your comments regarding this documentation. 
And do you think I should write documentation elsewhere in addition to this?

Reply:
@billsacks 
also in my TODO list from our recent conversation:
- Remake all the sfc datasets
- Run the full test suite

I will wait for the go-ahead from you guys, since I think this will be a time consuming step. By the way, is there a shortcut for generating ALL the sfc datasets at once?

Reply:
Also I just reread #648 and realized that I should address it next to get past this subsetting [error](https://github.com/ESCOMP/CTSM/issues/806#issuecomment-553644886) when using ocgis for our regridding. If so, then it prob. makes sense to wait and remake all the sfc datasets after that. Pls confirm.

Reply:
@slevisconsulting there is a Makefile "Makefile.data" that we use to send simultaneous jobs to create the standard datasets needed. Using "all" makes most of the standard ones outside of some of the high resolution ones. You could also name each of those to send them to make as well.

Reply:
This old post reminded me:
1) I made the new sfc datasets and landuse.timeseries files by running with the "all" option, 
**but they are sitting in** `/glade/scratch/slevis/temp_work/surfdata/branch_results`. Shall I copy them to `/glade/p/cesmdata/cseg/inputdata/lnd/clm2/surfdata_map` and rerun the rimport command for them?
2) When you consider this PR ready for merge, do you want me to run the full test-suites?